documentation.md

```diff
@@ -31,7 +31,7 @@ Cloud Haskell comprises the following components, some of which are complete,
 others experimental.
 
```

Comment:
We need to put among the lines that Cloud Haskell consists of 2 (3) big parts:

   * Core package `distributed-process` itself. Using this package on it's own may be enough to write any
      functionality. This package considered most stable. <And few more highflown words>  
   * Platform packages. Platform packages are built on the top of core packages and are intended to provide an implementation of patterns that are similar to OTP library (is it library?) in Erlang.

3 because we have also network-transport layer for communication between nodes (though this may be ommited).
Also there is d-p-simplelocalnet, and d-p-p2p that are not core, but neither platform.